docs/configurations-for-consensus-commit.md

```diff
@@ -0,0 +1,54 @@
+# Configurations for Consensus Commit
+
+Consensus Commit transaction manager is the default transaction manager type in Scalar DB.
+If you don't specify the `scalar.db.transaction_manager` property, or you specify `consensus-commit` for the property as follows, the Consensus Commit transaction manager is used.
+
+```properties
+scalar.db.transaction_manager=consensus-commit
+```
+
+The following sections describes the configurations for Consensus Commit.
+
+## Basic configurations
+
+The basic configurations for Consensus Commit is as follows:
+
+```properties
+# Default isolation level for ConsensusCommit. Either SNAPSHOT or SERIALIZABLE can be specified. SNAPSHOT is used by default.
+scalar.db.consensus_commit.isolation_level=SNAPSHOT
+
+# Default serializable strategy for ConsensusCommit transaction manager.
+# Either EXTRA_READ or EXTRA_WRITE can be specified. EXTRA_READ is used by default.
+# If SNAPSHOT is specified, this is ignored.
+scalar.db.consensus_commit.serializable_strategy=EXTRA_READ
```

Comment:
The comment explains but it’s a bit confusing to see both SNAPSHOT and EXTRA_READ are used in the configuration.
Should it explain each configuration separately?
How about making it a table-format like this? (can be simpler than this, though)
https://kafka.apache.org/documentation/#brokerconfigs

Reply:
I fixed it in the latest commit based on your feedback. Please take a look again when you have time!